### PR TITLE
[1LP][RFR] unifying flash message xpaths and fixing issue in hammer

### PR DESCRIPTION
**Pull request by izapolsk**
Many views have their own xpaths for FlashMessages. We need to unify that. 
Besides that there are flash messages stored in their own flash_text_div block in hammer. This also has to be sorted out
REQUIRES: https://github.com/RedHatQE/widgetastic.patternfly/pull/76

{{ pytest: cfme/tests/infrastructure/test_providers.py cfme/tests/automate/test_domain.py --long-running }}

### CODE REVIEW

File: cfme/base/ui.py [hunk 1]
```diff
@@ -1620,7 +1620,7 @@ def step(self):
 class AutomateImportExportBaseView(BaseLoggedInPage):
     # TODO This is currently overiding the base flash and should be renamed and efforts made
     # to update assocaited tests
-    flash = FlashMessages('div.import-flash-message')
```
Comment (izapolsk) [inline cfme/base/ui.py]: need to double check this. this change can potentially break some tests

**Comment by mshriver** [comment]
Included infra/test_providers, which does some flash message asserting, looking great. Might want to include some automate tests using the view that you're concerned with.

**Comment by izapolsk** [comment]
@mshriver, please review. PRT errors aren't related. I'll fix them in another PR.

**Comment by mshriver** [comment]
The updates to wt.patternfly have gotten much more in depth, so scope has changed a bit here. Need to make sure we're covering a good variety of flash message use in the tests selected for PRT.

**Comment by izapolsk** [comment]
@mshriver, there is one failure but it isn't related to FlashMessages changes. In addition, it appeared after code where previous error occurred .
I assume this is just flaky test and this error should go after restarting PRT.

**Pull request merged**